EasyPost/Exception.cs

```diff
@@ -66,5 +66,16 @@ public override string Message
     [Serializable]
     public class ClientNotConfigured : Exception
     {
+        public ClientNotConfigured(string message) : base(message)
+        {
+        }
+    }
+
+    [Serializable]
+    public class FilterFailure : Exception
+    {
+        public FilterFailure(string message) : base(message)
+        {
+        }
```

Comment:
What's all this?

Reply:
FilterFailure is a custom exception thrown specifically in the filtering functions, easier to specifically target, more descriptive to the end user about what the exact problem was (rather than a generic Exception)